Gemfile

```diff
@@ -103,6 +104,7 @@ gem 'activerecord-import', ">= 0.4.0"
 
 gem 'ruby-prof'
 group :development, :test do
+  gem 'byebug'
```

Comment:
This seems to be introducing a new dependency on `byebug` in an unrelated PR. I would suggest removing it for this contribution.

Reply:
This dependency will only be applied on :development and :test environments. Bundler will not install the gem if the environment is production (which is automatically set by capistrano).
As it is a usual development tool we added to improve development peace of the project. The weight of the gem is 82.5 KB which doesn't seem a problem for development envs.

Don't you think it's worth it? is it the case that this application has special constraints that we don't know?

Reply:
I see a few problems:

1. We have no idea of how byebug works, or what difference it makes to running the tests.
2. This dependency is added “on the side” of changing something completely different.
3. There is no documentation on why it was added, other than a very small commit message.

I have no idea of whether it’s a worthwhile addition or not, but I think it should probably be taken out of this PR. If you think it is a worthwhile addition, please propose it in an issue so that we can know more about the benefits.